front/Makefile

```diff
@@ -1,4 +1,5 @@
 export PATH := node_modules/.bin:$(PATH)
+NODE_MODULES := node_modules/.bin
```

Comment:
So, am I to leave `export PATH` as is, even though this change makes it useless? :)
Maybe there is another way to force OSX respect the `PATH` in the Makefiles?

Reply:
I've made a different workaround for OSX